src/pages/homepageAction.js

```diff
@@ -0,0 +1,16 @@
+const nav = document.getElementById("navbar");
+document.getElementById("menu").addEventListener("click",()=>{
+
+    const navStyle =  document.getElementById("navbar").style.display;
+    if(navStyle == "flex")
+    {
+        nav.style.display = "none"
+
+    }
+    else{
+        nav.style.display = "flex"
+
+    }
+   
+})
+
```

Comment:
hey saniya this is incorrect way of writing code in react better write this logic in the  component containing menu. I guess in Header.jsx file

Reply:
Done changes